Provider/src/FirebirdSql.EntityFrameworkCore.Firebird/Metadata/FbValueGenerationStrategy.cs

```diff
@@ -18,8 +18,9 @@
 namespace FirebirdSql.EntityFrameworkCore.Firebird.Metadata
 {
 	public enum FbValueGenerationStrategy
-    {
+	{
+		None,
 		SequenceTrigger,
-		IdentityColumn,
-    }
+		IdentityColumn
```

Comment:
Missing trailing comma.

Reply:
In being the last identifier, I did not need this, but I'll put it now!

Thanks!

Reply:
I know. But I kind of like putting it there anyway. It helps future changes of the code.